example/src/main/java/com/transferzero/test/Application.java

```diff
@@ -7,11 +7,12 @@
 import com.transferzero.sdk.api.SendersApi;
 import com.transferzero.sdk.api.TransactionsApi;
 import com.transferzero.sdk.model.*;
-import java.time.LocalDate;
+import org.joda.time.LocalDate;
```

Comment:
Seems like J8 SDK example is using `time.LocalDate` where the J7 SDK example is using `joda.time.LocalDate`. Unless there's a specific reason for changing this here and we know it's not going to break any of the examples, I would keep each project as they are in master.

Reply:
> "Joda-Time is the de facto standard date and time library for Java prior to Java SE 8"

[Explains this](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=2ahUKEwi-7vSSs8HlAhWDShUIHQiAAvEQFjAAegQICRAC&url=https%3A%2F%2Fwww.joda.org%2Fjoda-time%2F&usg=AOvVaw3bu2XuCBvlskQZqKdUtRQD), so we should keep this as `java.time` and use joda for the J7 SDK

Reply:
Yes, this is the main difference between the J7 and J8 version. J7 should use Joda, while J8 should use java.time